dev-libs/libucl/libucl-0.9.2.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY=" Universal configuration library parser"
+DESCRIPTION="Libucl is a parser and C API to parse and generate ucl objects."
+HOMEPAGE="https://github.com/vstakhov/libucl"
+COPYRIGHT="2013-2014 Vsevolod Stakhov"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/vstakhov/libucl/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="f63ddee1d7f5217cac4f9cdf72b9c5e8fe43cfe5725db13f1414b0d8a369bbe0"
+SOURCE_FILENAME="libucl-$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="7.2.0"
+libVersionCompat="$libVersion compat >= 7"
```

Comment:
Missed a few (almost there), thanks for your patience!

```suggestion
libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Reply:
updated